### PR TITLE
Testing out java 11 readiness

### PR DESCRIPTION
**Pull request by res0nance**

### CODE REVIEW

**Comment by res0nance** [comment]
Looks like this is hard for me to test myself since it requires write permissions

**Comment by res0nance** [comment]
@jvz do you mind re-reviewing this change?

File: pom.xml [hunk 1]
```diff
@@ -76,8 +76,7 @@ THE SOFTWARE.
     <properties>
         <revision>1.43</revision>
         <changelist>-SNAPSHOT</changelist>
-        <powermock.version>1.6.3</powermock.version>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.89.4</jenkins.version>
```
Comment (jvz) [inline pom.xml]: This is still pretty old. @thoulen any preferences on a minimum version?

Comment (res0nance) [reply pom.xml]: @jvz I updated it to `2.121.1` along with a bunch of test deps to make the java 11 tests pass

**Pull request merged**